arlo-client/src/components/CreateAudit.test.tsx

```diff
@@ -42,17 +43,152 @@ describe('CreateAudit', () => {
     expect(container).toMatchSnapshot()
   })
 
-  it('calls the /election/new endpoint', async () => {
+  it('calls the /election/new endpoint for nonauthenticated user', async () => {
     apiMock.mockImplementation(async () => ({ electionId: '1' }))
     const { getByText } = render(<CreateAudit {...routeProps} />)
 
     fireEvent.click(getByText('Create a New Audit'), { bubbles: true })
 
     await wait(() => {
       expect(apiMock).toBeCalledTimes(1)
-      expect(apiMock.mock.calls[0][0]).toBe('/election/new')
+      expect(apiMock).toHaveBeenNthCalledWith(1, '/election/new', {
+        method: 'POST',
+        body: JSON.stringify({}),
+      })
+      expect(historySpy).toBeCalledTimes(1)
+      expect(historySpy).toHaveBeenNthCalledWith(1, '/election/1')
+    })
+  })
+
+  it('calls the /election/new endpoint for authenticated user', async () => {
+    apiMock
+      .mockImplementationOnce(async () => ({
+        type: 'audit_admin',
+        name: 'Joe',
+        email: 'test@email.org',
+        jurisdictions: [],
+        organizations: [
+          {
+            id: 'org-id',
+            name: 'State',
+            elections: [],
+          },
+        ],
+      }))
+      .mockImplementationOnce(async () => ({ electionId: '1' }))
+    const { getByText } = render(
+      <AuthDataProvider>
+        <CreateAudit {...routeProps} />
+      </AuthDataProvider>
+    )
+
+    await wait(() => {
+      expect(apiMock).toBeCalledTimes(1)
+      expect(apiMock).toHaveBeenNthCalledWith(1, '/auth/me')
+    })
+
+    fireEvent.click(getByText('Create a New Audit'), { bubbles: true })
+
+    await wait(() => {
+      expect(apiMock).toBeCalledTimes(2)
+      expect(apiMock).toHaveBeenNthCalledWith(2, '/election/new', {
+        method: 'POST',
+        body: JSON.stringify({ organization: 'org-id' }),
```

Comment:
this should probably be `organization_id`

Reply:
@benadida Done!